src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+}
+
+@section Back {
+    <a href="/ProfessionalReferral/ConnectFamilyToServiceStart?id=@Model.Id&name=@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<form method="post">
+    <input asp-for="@Model.Name" type="hidden">
+    <input asp-for="@Model.Id" type="hidden">
+    <div class="govuk-form-group">
+        <div class="govuk-form-group ">
+
+            @if (!Model.ValidationValid)
+            {
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        There is a problem
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li>
+                                <a href="#service-name" data-testid="enter-service-name-error">Select whether you have permission from the family to share details.</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            }
+
+            <fieldset class="govuk-fieldset">
+                <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                    <h1 class="govuk-fieldset__heading">
+                        Consent
```

Comment:
Looks like the page content doesn't match the prototype.

Can you check and update if necessary please.

Reply:
Fixed